drivers/usb/udc/udc_dwc2_vendor_quirks.h

```diff
@@ -79,6 +79,46 @@ static inline int stm32f4_fsotg_disable_phy(const struct device *dev)
 	return 0;
 }
 
+static inline int stm32f4_fsotg_init_caps(const struct device *dev)
+{
+	const struct udc_dwc2_config *config = dev->config;
+	int err;
+
+	/* Just assume it is always continuous for that platform. */
```

Comment:
I believe `this platform` (instead of `that platform`) would be appropriate here.